tools/testing/kata-webhook/deploy/kata-webhook-config-map.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
May just define this configmap in https://github.com/kata-containers/kata-containers/blob/main/tools/testing/kata-webhook/deploy/webhook.yaml as another resource separated by `---`

Reply:
Done.